src/cosalib/aws.py

```diff
@@ -111,6 +112,20 @@ def aws_run_ore_replicate(build, args):
 
 @retry(reraise=True, stop=stop_after_attempt(3))
 def aws_run_ore(build, args):
+     # Capture any input from image.yaml
+    image_yaml = flatten_image_yaml(
+        '/usr/lib/coreos-assembler/image-default.yaml',
+        flatten_image_yaml('src/config/image.yaml')
+    )
+    if 'aws-imdsv2-only' in image_yaml and image_yaml['aws-imdsv2-only'] :
+        ore_args.extend(['--imds-support'], "v2.0")
+    if 'aws-volume-type' in image_yaml:
+        ore_args.extend(['--volume-type', image_yaml['aws-volume-type']])
+
+        # still need logic for determining architecture.
+    if 'aws-x86-boot-mode' in image_yaml:
+        ore_args.extend(['--boot-mode', image_yaml['aws-x86-boot-mode']])
```

Comment:
Should this be:
```suggestion
        ore_args.extend(['--x86-boot-mode', image_yaml['aws-x86-boot-mode']])
```

Reply:
Hmm, it is. I was under the impression that the ore_args.extend used the aws's api? so we had to be specific to what ever AWS needed? i.e '--boot-mode'